pattern-categorization/gl/innersource-program-mind-map.html

```diff
@@ -27,6 +27,6 @@
n propietario/mantenedor","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/core-team\">Equipo Central</a>","children":[],"payload":{"tag":"h5","lines":"96,97"}},{"content":"<a href=\"https://patterns.innersourcecommons.org/p/group-support\">Soporte de Grupo</a>","children":[],"payload":{"tag":"h5","lines":"98,99"}}],"payload":{"tag":"h4","lines":"94,95"}}],"payload":{"tag":"h3","lines":"68,69"}},{"content":"Desafíos entre Entidades Legales","children":[{"content":"Preocupación por responsabilidades legales o contabilidad entre empresas","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/innersource-license\">Licencia InnerSource</a>","children":[],"payload":{"tag":"h5","lines":"104,105"}}],"payload":{"tag":"h4","lines":"102,103"}}],"payload":{"tag":"h3","lines":"100,101"}}],"payload":{"tag":"h2","lines":"32,33"}},{"content":"Crecer","children":[{"content":"Desafíos de Descubrimiento","children":[{"content":"No se pueden encontrar proyectos coincidentes","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/gig-marketplace\">Mercado de Gigs</a>","children":[],"payload":{"tag":"h5","lines":"112,113"}},{"content":"<a href=\"https://patterns.innersourcecommons.org/p/innersource-portal\">Portal InnerSource</a>","children":[],"payload":{"tag":"h5","lines":"114,115"}}],"payload":{"tag":"h4","lines":"110,111"}},{"content":"Difícil encontrar proyectos activos","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/repository-activity-score\">Puntuación de Actividad del Repositorio</a>","children":[],"payload":{"tag":"h5","lines":"118,119"}}],"payload":{"tag":"h4","lines":"116,117"}}],"payload":{"tag":"h3","lines":"108,109"}}],"payload":{"tag":"h2","lines":"106,107"}},{"content":"Escalar","children":[{"content":"Desafíos de Autoeducación/Mejora","children":[{"content":"No estar al tanto de las mejores prácticas de InnerSource","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/maturity-model\">Modelo de Madurez</a>","children":[],"payload":{"tag":"h5","lines":"126,127"}}],"payload":{"tag":"h4","lines":"124,125"}},{"content":"Falta de conocimiento de código abierto","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/document-your-guiding-principles\">Documenta tus Principios Rectores</a>","children":[],"payload":{"tag":"h5","lines":"130,131"}}],"payload":{"tag":"h4","lines":"128,129"}}],"payload":{"tag":"h3","lines":"122,123"}},{"content":"Desafíos Técnicos","children":[{"content":"Aumento de la sobrecarga de mantenimiento","children":[{"content":"<a href=\"https://patterns.innersourcecommons.org/p/extensions-for-sustainable-growth\">Extensiones para el Crecimiento Sostenible</a>","children":[],"payload":{"tag":"h5","lines":"136,137"}}],"payload":{"tag":"h4","lines":"134,135"}}],"payload":{"tag":"h3","lines":"132,133"}}],"payload":{"tag":"h2","lines":"120,121"}}],"payload":{"tag":"h1","lines":"0,1"}},null)</script>
```

Comment:
IMHO the PR should only cover changes and translations for Spanish content, otherwise it could be tricky to follow up. I would like to skip any change not related to Spanish, and open an issue (and PR) to propose the changes in the other languages. Maybe someone from the core team could double check and confirm it or not this approach.

Reply:
Addressed in the related thread.